### PR TITLE
Silent authentication

### PR DESCRIPTION
**Pull request by batalla3692**
Add silent authentication functionality
Now you can pass `prompt` parameter in `redirect_to` path, and `authorize_params` method will add it to the authorization params.

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@batalla3692 - Thank you for the PR and apologies for the delay in response here (did not have notifications setup but I do now).

I have a quick piece of feedback that I'll post as a review, otherwise this looks fine.

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ Gemfile.lock
 .#*
 .env
 log/
-tmp/
\ No newline at end of file
+tmp/
+.idea
```
Comment (joshcanhelp) [inline .gitignore]: IDE files should be in a global .gitignore:

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Comment by batalla3692** [comment]
Thanks @joshcanhelp, I remove that rule from **.gitignore**

**Comment by joshcanhelp** [comment]
Thanks @batalla3692! I'll merge this now and release it as soon as possible. We're working on getting ID token validation in for the next release so it will be bundled with that.

**Pull request merged**